x-pack/plugin/sql/qa/src/main/resources/examples/example.csv-spec

```diff
@@ -23,4 +23,45 @@ foo,2.5,3
 bar,3.5,4
 tar,4.5,5
 ;
-// repeat the above
\ No newline at end of file
+// repeat the above
+
+
+// Test failure example:
+// org.elasticsearch.xpack.sql.qa.single_node.JdbcCsvSpecIT > test {pivot.testAverageWithOneValueAndLimit} FAILED
+//    java.lang.AssertionError: Elasticsearch [JdbcResultSet:row 4:cursor size 2:[languages<type=[BYTE]>, 'F'<type=[DOUBLE]>]] still has data after [3] entries:
+//    3                    | 53660.0
+//        at __randomizedtesting.SeedInfo.seed([81AF0740AC9F7D95:9FB389A0263106D]:0)
+//        at org.elasticsearch.xpack.sql.qa.single_node.JdbcCsvSpecIT.pivot.testAverageWithOneValueAndLimit(pivot.csv-spec:95)
```

Comment:
It's worth pointing out that the stacktrace contains the file and line of the test that fails, just in case the assignee hasn't noticed that already.